docs/developer.md

```diff
@@ -157,7 +157,7 @@ Check your linting.
  kubectl apply -f ./deploy/rbac.yaml
  kubectl apply -f ./deploy/operator.yaml
  ```
-- Run the chaos by following the [Litmus Docs](https://docs.litmuschaos.io/docs/getstarted/#install-chaos-experiments)
```

Comment:
Provide this link: https://v1-docs.litmuschaos.io/docs/getstarted#install-chaos-experiments

Reply:
Done @ispeakc0de